device/device_api.h

```diff
@@ -7,4 +7,8 @@
 #pragma once
 #include "device/tt_device.h"
 #include "device/driver_atomics.h"
-#include "device/tt_emulation_device.h"
\ No newline at end of file
+#include "device/tt_emulation_device.h"
+
+#ifdef TT_METAL_SIMULATOR_EN
```

Comment:
should this be named `TT_UMD_SIMULATOR_EN`?

Reply:
removed